tests/e2e/esm/index.test.ts

```diff
@@ -1108,4 +1109,60 @@ describe('e2e esm tests', async () => {
       }
     });
   });
+
+  // Skipping due to inconsistent results with spun up CU
```

Comment:
these work when I hit cu.ardrive or use AO infra. but using our local CU this causes a false negative. we have this covered in unit test though